app/src/main/kotlin/com/proxerme/app/helper/ScreenHelper.kt

```diff
@@ -0,0 +1,11 @@
+package com.proxerme.app.helper
+
+import android.content.Context
+import com.proxerme.app.R
+
+object ScreenHelper {
+
+    fun isTablet(context: Context): Boolean {
```

Comment:
We already have a function for this [here](https://github.com/proxer/ProxerAndroid/blob/master/app/src/main/kotlin/com/proxerme/app/util/DeviceUtils.kt#L20). Please use that one.